### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by guedou**
This PR proposes an enhanced `README.md` with embedded examples and links to the notebooks.

The Scapy logo is currently in `doc/scapy_logo.png`. This raises some questions:
- shall we put it in anothe repository ?
- shall we reduce its size ?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,59 +1,119 @@
...
+unit tests, attacks or network discovery (it can replace `hping`, 85% of `nmap̀`,
+`arpspoof`, `arp-sk`, `arping`, `tcpdump`, `wireshark`, `p0f`, etc.). It also
+performs very well at a lot of other specific tasks that most other tools can't
+handle, like sending invalid frames, injecting your own 802.11 frames, combining
+techniques (VLAN hopping+ARP cache poisoning, VoIP decoding on WEP protected
+channel, ...), etc.
+
+## Hands-on
+
+###### Interactive shell
```
Comment (p-l-) [inline README.md]: Not sure if that matters in the markdown rendering, but is there a reason for using level 6 `######` titles rather than level 3 `###`?

**Comment by p-l-** [comment]
@guedou we can keep the logo in the repository (see with its author about license maybe?), with a good enough resolution. We can add the logo to the README.md with a `` HTML tag (see IVRE repository for example), maybe we can use the tag to give force a small width?

**Comment by gpotter2** [comment]
Very nice logo !

File: README.md (hunk 1 above)
Comment (guedou) [reply README.md]: I could not make it work with `### Hands-on` only =)

**Comment by guedou** [comment]
@p-l- we bought the logo, so we own it. I don't know which license should be used in this case. The markdown already uses the HTML trick to change the image with to 200px. I wonder if the `doc/scapy_logo.png` size should be reduced to 200px.

**Comment by p-l-** [comment]
@guedou I think in that case we can leave the high resolution version in the repository, and let the web client resize it when displaying the README file.

**Pull request merged**